src/lib/validator-string.ts

```diff
@@ -6,7 +6,7 @@ export default class ValidatorString implements IValidator {
   }
 
   format(input : string) : string {
-    input = input.replace(/[^a-zA-Z0-9 ()@./\-']/g, '');
+    input = input.replace(/[^a-zA-Z0-9 ()@./\-'éôîêàèùçœ]/gu, '');
```

Comment:
Do we care about upper case?  Should we use something like `A-Za-zÀ-ÖØ-öø-ÿ`  [source](https://stackoverflow.com/a/26900132)?